index.html

```diff
@@ -34,6 +34,187 @@ <h2>I'm a Software Developer</h2>
+              </p>
+              <button>Get my resume</button>
+            </section>
+
+            <section class="lists">
+              <hr>
+
+              <section class="attach-icon">
+                <img src="./logos/Rectangle 55.png" alt="Rectangle">
+                <h3>Language</h3>
+              </section>
+
+              <ul>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">Ruby</a></li>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">CSS</a></li>
+              </ul>
+
+              <section class="attach-icon">
+                <img src="./logos/Square.png" alt="Rectangle">
+                <h3>Frameworks</h3>
+              </section>
+
+              <ul>
+                <li><a href="#">React.js</a></li>
+                <li><a href="#">Ruby on Rails</a></li>
+                <li><a href="#">RSpec</a></li>
+                <li><a href="#">CapyBara</a></li>
+              </ul>
+              
+              <section class="attach-icon">
+              <img src="./logos/Circle.png" alt="Rectangle">
+              <h3>Skills</h3>
+              </section>
+
+              <ul>
+                <li><a href="#">Database Management</a></li>
+                <li><a href="#">Version Control</a></li>
+                <li><a href="#">CLI</a></li>
+                <li><a href="#">API Design</a></li>
+                <li><a href="#">Web Development</a></li>
+              </ul>
+            </section>
+          </section>
+
+          <section class="interesting">
+            <h4>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+            <input type="text" class="full-name" name="full-name" placeholder="Full name">
+            <hr class="end-name">
+            <input type="text" class="email-address" name="email-address" placeholder="Email address">
+            <hr class="end-address">
+            <textarea class="text-area" name="text-area" rows="4" cols="50" placeholder="Write me something..."></textarea>
+            <button>Get in touch</button>
+          </section>
+
+          <hr class="end-page">
+          
+          <footer class="footer">
+            <ul>
+              <li><a class ="init-icon" href="https://twitter.com/anyarsencarta"><img src="./logos/Cat Deep.png" alt="Linkedin"></a></li>
+              <li><a href="https://linkedin.com/anyarsencarta"><img src="./logos/Linkedin Deep.png" alt="Linkedin"></a></li>
+              <li><a href="https://twitter.com/anyarsencarta"><img src="./logos/Hand Deep.png" alt="Linkedin"></a></li>
+              <li><a href="https://twitter.com/anyarsencarta"><img src="./logos/Bird Deep.png" alt="Twitter"></a></li>
+              <li><a href="https://twitter.com/anyarsencarta"><img src="./logos/M Deep.png" alt="Linkedin"></a></li>
+            </ul>
+            <hr>
+          </footer>
+        </section>
```

Comment:
- [ ] Please remove this section because we are going to work on the Contact form in another milestone, remember that you have to create a new branch, if you don't remove this part now you won't be able to comment on the changes in the new branch to create the PR for review.